completion/available/bash-it.completion.bash

```diff
@@ -65,7 +65,7 @@ _bash-it-comp()
   prev="${COMP_WORDS[COMP_CWORD-1]}"
   chose_opt="${COMP_WORDS[1]}"
   file_type="${COMP_WORDS[2]}"
-  opts="disable enable help migrate search show update version"
+  opts="disable enable help migrate search show update version reload"
```

Comment:
Could you please add the new `reload` option in alphabetical order, i.e. between `migrate` and `search`?